docs/source/API/core/execution_spaces.rst

```diff
@@ -327,3 +329,5 @@ Non Member Facilities
 * ``template<class MS> struct is_execution_space;``: typetrait to check whether a class is a execution space.
 
 * ``template<class S1, class S2> struct SpaceAccessibility;``: typetraits to check whether two spaces are compatible (assignable, deep_copy-able, accessable). (see |KokkosSpaceAccessibility|_)
+
+* ``bool operator==(const S& lhs, const S& rhs)``: tests whether the two space instances (of the same type) are identical.
```

Comment:
What's `S`?  Should it read `ExecutionSpaceConcept`?

Reply:
Even though it's declared inside the ExecutionSpaceConcept class, it's a non-member friend function. So I was trying to follow the pattern of the other non-member features like the one above which uses S1,S2.

Reply:
I could change the type here to ``execution_space`` or ``ExecutionSpaceConcept`` or anything else if you prefer that

Reply:
I think `execution_space`

Reply:
Yeah, I actually like that better now too. I just pushed the change